proc_dash/app.py

```diff
@@ -230,6 +264,34 @@
 )
 
 
+@app.callback(
+    [
+        Output("dataset-name-modal", "is_open"),
+        Output("summary-title", "children"),
+        Output("dataset-name-input", "value"),
```

Comment:
Curious, why do you update the input here?

Reply:
Ah, good question. This is to reset the input value so that if the user has uploaded a file and set a dataset name, and then wants to upload a _different_ file, the default name will be `"Dataset"` again as opposed to the name of the previously uploaded dataset. 🙂